<?xml version="1.0" encoding="utf-8"?>
<launch>
  <!-- Console launch prefix -->
  <arg name="launch_prefix" default=""/>

  <!-- Config and weights folder. -->
  <rosparam file="$(find robot_vision_openvino)/config/ros.yaml" command="load" ns="interactive_face_ros" />
  <rosparam file="$(find robot_vision_openvino)/config/interactive_face.yaml" command="load" ns="interactive_face_ros" />
  <!-- Start openvino yolo node -->
  <node pkg="robot_vision_openvino" type="interactive_face_ros" name="interactive_face_ros" output="screen" launch-prefix="$(arg launch_prefix)">
  </node>

 <!--<node name="republish" type="republish" pkg="image_transport" output="screen" 	args="compressed in:=/front_camera/image_raw raw out:=/camera/image_raw" /> -->
</launch>
